### ISSUE TITLE
[Feature Request] A pen cursor would be better than the irritating crosswire cursor! And need the visibility of the pen cursor while writing

### ISSUE THREAD

**Issue by MiltonBalaOfficial** [OpenBoard-org/OpenBoard#1061]
(1) The crosswire cursor is very irritating, Please change it to a pen cursor if possible or provide a option to choose.
(2) For online class, the tracking of the position of where the writing is done is very important, Tracking could be more pleasant if the pen cursor remain visible while writing. Please provide an option to make it happen.
thanks

**Comment by econmaths** [comment]
I agree, for instance, xournal++ pen cursor is nice and usefull on the screen.

**Comment by MiltonBalaOfficial** [comment]
I think, many users ( #795 ) .want a pen cursor instead of cross-hair cursor. I also feel the visibility of the cursor while writing! ...* X

I found a way. I have changed cross-hair cursor into a pen cursor from the control panel. But the visibility of the cursor while writing is beyond my ability. The developers may do something!

**Comment by RiyadunNabi** [comment]
**I strongly support this feature request. Persistent cursor visibility would greatly enhance usability for presentations and teaching.**

**Comment by MiltonBalaOfficial** [comment]
> I agree, for instance, xournal++ pen cursor is nice and usefull on the screen.

Thanks, I even don't know about Xournal++, now I am using it. And it is awesome! I love its customizability and the plugin option is great!

I have converted all of my openboard document to xournal document with the help of a parser!
